tcms/testcases/static/testcases/js/get.js

```diff
@@ -200,20 +200,62 @@ $(document).ready(function() {
             processing: '<div class="spinner spinner-lg"></div>',
             zeroRecords: "No records found"
         },
-        order: [[ 0, 'asc' ]],
+        order: [[0, 'asc']],
     });
 
     // remove plan button
-    plans_table.on('draw', function() {
-        $('.remove-plan').click(function() {
+    plans_table.on('draw', function () {
+        $('.remove-plan').click(function () {
             var tr = $(this).parents('tr');
 
-            jsonRPC('TestPlan.remove_case', [$(this).data('pk'), case_id], function(data) {
+            jsonRPC('TestPlan.remove_case', [$(this).data('pk'), case_id], function (data) {
```

Comment:
@atodorov please ignore all these reformats. this is the right way the code should look like and I already applied the changes in separate PR - https://github.com/kiwitcms/Kiwi/pull/2474. let's just merge 2474 first and after that I will rebase this one, and this diff will go away